codegen-core/common-test-models/constraints.smithy

```diff
@@ -10,7 +10,6 @@ use smithy.framework#ValidationException
 @title("ConstraintsService")
 service ConstraintsService {
     operations: [
-        // TODO Rename as {Verb}[{Qualifier}]{Noun}: https://github.com/awslabs/smithy-rs/pull/1342#discussion_r980936650
```

Comment:
I removed this because I couldn't think of any good names. The academic names of the operations are clear enough.